autoload/neomake/utils.vim

```diff
@@ -189,8 +189,7 @@ function! neomake#utils#MakerFromCommand(command) abort
     " XXX: use neomake#utils#ExpandArgs and/or remove it.
     "      Expansion should happen later already!
     " NOTE: useful when calling it from cmdline..
-    let command = substitute(a:command, '%\(:[a-z]\)*',
-                           \ '\=expand(submatch(0))', 'g')
+    let command = neomake#utils#ExpandArgs([a:command])[0]
```

Comment:
Ok, makes sense like this here I assume, since it won't be expanded later.
Should remove the `XXX`/`NOTE` here though.